domains/bongani.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Developer Portfolio",
+    "repo": "https://github.com/cyneweard-zw/portfolio",
+    "owner": {
+        "username": "cyneweard-zw",
+        "email": "cyneweard@outlook.com",
+        "twitter": "@cyneweard_zw"
+    },
+    "record": {
+        "CNAME": "cyneweard-zw.github.io"
```

Comment:
You should complete your website first, right now I see a template website with still the default text like "Contact information or a form goes here."